packages/edit-site/src/components/page-pages/index.js

```diff
@@ -12,20 +12,20 @@ import { useState, useMemo, useCallback, useEffect } from '@wordpress/element';
 import { dateI18n, getDate, getSettings } from '@wordpress/date';
 import { privateApis as routerPrivateApis } from '@wordpress/router';
 import { useSelect, useDispatch } from '@wordpress/data';
+import {
+	DataViews,
+	ENUMERATION_TYPE,
+	VIEW_LAYOUTS,
+	OPERATOR_IN,
+	LAYOUT_GRID,
+	LAYOUT_TABLE,
+} from '@wordpress/dataviews';
```

Comment:
Personally, I'm not a big fan of exporting constants from package but I can live with it for now.

Reply:
There are a few things here:

- The string constants can be removed https://github.com/WordPress/gutenberg/pull/56754
- The `VIEWS_LAYOUT` is used for two things 
  - Detect supports, which also can be removed if we go with this API for the list view/preview https://github.com/WordPress/gutenberg/pull/56746
  - Use the icons in the sidebar. What do we do with this one? Export the icons directly or provide a utility such as `getIconForViewType( viewType )`?